openownership/data-standard
# 40 interest types

# Overview

- What does this pull request do?
Implements the changes to trust-like legal arrangements as discussed in #40 
- How can a reviewer test or examine your changes?
Sanity check the changes proposed in the issue 
Check that it faithfully implements the changes in the issue
- Who is best placed to review it?

(Closes/Relates to) issue: #40 

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html
@StephenAbbott for info

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html
